app/Actions/Place/GetPlaceItem/GetPlaceItemPresenter.php

```diff
@@ -61,6 +61,7 @@ public function present(GetPlaceItemResponse $placeResponse): array
         $result['localization'] = $this->localizationPresenter->presentCollection($place->localization);
         $result['category'] = $this->categoryPresenter->present($place->category);
         $result['category']['tags'] = $this->tagsPresenter->presentCollection($place->category->tags);
+        $result['myRating'] = (int)$place->ratings->where('user_id', $placeResponse->getUser()->id)->avg('rating');
```

Comment:
not sure if the right place

Reply:
We have method for that in ratingRepository